middlewares/validators/qrCodeAuth.js

```diff
@@ -0,0 +1,22 @@
+const joi = require("joi");
+
+const validateAuthStatus = async (req, res, next) => {
+  const schema = joi
+    .object()
+    .strict()
+    .keys({
+      authorization_status: joi.string().valid("AUTHORIZED", "REJECTED", "NOT_INIT").optional(),
```

Comment:
Is `optional()` needed here?
As `authorization_status` request param needs to be present for this route to be called

Reply:
I kept this as optional, because if no status is passed, we are setting authorization_status as "NOT_INIT" by default. @shubham-y

Reply:
the authorization_status is optional, route: "/qr-code-auth/authorization_status/:authorization_status?" @shubham-y

Reply:
is it optional in the contract, i don't think it is, can u check once?

Reply:
I used optional, so that when we call the patch route without any status, by default it will set the status to "NOT_INIT".

Reply:
can we confirm, this with @shreya-mishra or @heyrandhir ?

Reply:
> the authorization_status is optional, route: "/qr-code-auth/authorization_status/:authorization_status?" @shubham-y

Ohh right, I wasn't aware that it's possible to do this way in express.

Reply:
@fakhruddinkw So have we decided to remove optional?

Reply:
@shubham-y Yes i had a discussion regarding this with @prakashchoudhary07 , and keeping the route as well as the schema optional is not needed.